spec/dummy/config/database.yml

```diff
@@ -5,21 +5,26 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
   pool: 5
-  timeout: 5000
+  host: localhost
```

Comment:
Does anyone use a non-localhost database when running locally?

The travis configuration is handled via the database.travis.yml.